examples/stable/test/property/ic_api/json/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "json",
```

Comment:
Get rid of the name and version. Also the dependencies and devDependencies should look like the other package.json files, you shouldn't have to install so many things.